hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3ABlockOutputStream.java

```diff
@@ -586,21 +586,21 @@ private int putObject() throws IOException {
             builder.putOptions,
         false);
 
-          // TODO: You cannot currently add progress listeners to requests not via the TM.
-          // There is an open ticket for this with the SDK team. But need to check how important
-          // this is for us?
-    //    BlockUploadProgress callback =
-    //        new BlockUploadProgress(
-    //            block, progressListener,  now());
+        BlockUploadProgress progressCallback =
+            new BlockUploadProgress(
+                block, progressListener, now());
     // putObjectRequest.setGeneralProgressListener(callback);
```

Comment:
remove commented out code - or do we think this could be reverted in the future? (more comments needed in that case)